### PR TITLE
fix for config links

### PR DESCRIPTION
**Pull request by wyntonfranklin**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

fix for issue #5400

### CODE REVIEW

**Comment by wyntonfranklin** [comment]
@j0k3r didn't see any other areas in config where this could be applicable.

**Review by j0k3r — APPROVED** [review]
Thanks for the contribution!

File: src/Wallabag/CoreBundle/Resources/views/Config/index.html.twig [hunk 1]
```diff
@@ -45,7 +45,7 @@
                                     {{ form_label(form.config.reading_speed) }}
                                     <p>
                                         {{ 'config.form_settings.reading_speed.help_message'|trans }}
-                                        <a href="https://wallabag.github.io/myreadspeed/">myreadspeed</a>
+                                        <a target="_blank" href="https://wallabag.github.io/myreadspeed/">myreadspeed</a>
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/views/Config/index.html.twig:48]: `rel="noopener noreferrer"` would have been nice.

Comment (wyntonfranklin) [reply src/Wallabag/CoreBundle/Resources/views/Config/index.html.twig:48]: :( now u tell me

**Comment by nicosomb** [comment]
Just a little post, I'm not fan of `_blank` 😬

**Comment by wyntonfranklin** [comment]
@nicosomb what's the better way

**Comment by nicosomb** [comment]
The better way is to let the user decide.

**Comment by tcitworld** [comment]
@nicosomb The change is not in the article body, just in two links in the config section.

**Pull request merged**